<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>winWeaTool</class>
 <widget class="QMainWindow" name="winWeaTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Space Switcher Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>511</width>
      <height>701</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_weap">
     <attribute name="title">
      <string>Weapons Control</string>
     </attribute>
     <widget class="QPushButton" name="pushBut_r_h_driver_l_attach">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>380</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>R Hand Driver /
 L Hand Attached</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushBut_weap_driver_r_hand_attach">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>480</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Weapon Driver /
 only R Hand Attached</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushBut_break_r_h_const">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>290</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Break R Hand
 Constraint</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushBut_r_hand_constr">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>220</y>
        <width>191</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Constraint R Hand to weapon Driver</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushBut_l_h_driver_r_attach">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>480</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>L Hand Driver /
 R Hand Attached</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushBut_r_h_driver_l_free">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>430</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string> R Hand Driver /
 L Hand Free</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushBut_weap_driver_l_hand_attach">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>430</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Weapon Driver /
 only L Hand Attached</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_choose_project">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose  Project</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboB_choose_project">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushBut_break_wea_constr">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>280</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Break Weapon Constraint</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushBut_l_h_driver_r_free">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>530</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string> L Hand Driver /
 R Hand Free</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushBut_weap_driver_hands_attach">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>380</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Weapon Driver /
 L and R Hands Attached</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_choose_weapon">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose Weapon</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushBut_ref_weap">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>170</y>
        <width>391</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Reference Weapon</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushBut_constrain_weap">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>220</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Constraint Weapon to Char Driver</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_chars_drivers_presets">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>360</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Char Driver Presets</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_weapon_drivers_presets">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>360</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Weapon Driver Presets</string>
      </property>
     </widget>
     <widget class="QLabel" name="lab_file_2_ref">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>File to reference:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushBut_break_l_h_const">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>290</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Break L Hand
 Constraint</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEd_file_ref">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>110</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushBut_l_hand_constr">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>250</y>
        <width>191</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Constraint L Hand to weapon Driver</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboB_choose_weap">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushBut_ressete_all_attr">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>590</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Resete All Attributes
 and Constraints</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_body">
     <attribute name="title">
      <string>Body Controls</string>
     </attribute>
     <widget class="QComboBox" name="comboBCharacters">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_charsnamespace">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Character</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButIkarm_l">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>340</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>IKArm_L</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButIkarm_r">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>IKArm_R</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushBut_Head">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>130</y>
        <width>101</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>FKHead_M</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButshoulder_r">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>FKShoulder_R</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButshoulder_l">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>250</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>FKShoulder_L</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButRoot">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>490</y>
        <width>131</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>FKroot_M</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushBut_Scap_l">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>220</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>FKScapula_L</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushBut_Scap_r">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>220</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>FKScapula_R</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>547</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
